pkg/cmdutil/userio/wizard/wizard.go

```diff
@@ -131,12 +135,29 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		log.Debug().Msgf("Wizard: Received [%T]", msg)
 		if messageLen := len(m.messageChannel); messageLen > 0 {
 			log.Debug().Msgf("Wizard: Message channel still has %d items, postponing shutdown", messageLen)
-			return m, tea.Sequence(updateListener, func() tea.Msg { return doneMsg(true) })
+			return m, updateListener
 		} else {
 			m.markLatestTaskComplete()
 			m.quitting = true
 			return m, tea.Quit
 		}
+	case errorMsg:
+		log.Debug().Msgf("Wizard: Received [%T]", msg)
+		if messageLen := len(m.messageChannel); messageLen > 0 {
+			log.Debug().Msgf("Wizard: Message channel still has %d items, postponing shutdown", messageLen)
+			return m, updateListener
+		} else {
+			m.quitting = true
+			return m, tea.Sequence(
+				func() tea.Msg {
+					return updateCurrentTaskCompletedTitle{
+						title:  msg.Error(),
```

Comment:
Suggestion: if we use an error just to display a message, we can pass string directly, without wrapping it as an error. I think it's more convenient